src/rust/catloop/mod.rs

```diff
@@ -388,7 +390,7 @@ impl CatloopLibOS {
                     }
                 }
 
-                runtime.free_queue::<CatloopQueue>(qd);
+                runtime.free_queue::<CatloopQueue>(&qd).expect("queue should exist");
```

Comment:
To follow our guidelines, could you add a comment here explaining why it is safe to call `expect()` here?